auxx/src/Command/Update.hs

```diff
@@ -126,13 +131,65 @@ propose sendActions ProposeUpdateParams{..} = do
                         udata
                         (mkAttributes ())
                         ss
-            if null ccPeers
-                then putText "Error: no addresses specified"
-                else do
-                    submitUpdateProposal (immediateConcurrentConversations sendActions ccPeers) ss updateProposal
-                    let id = hash updateProposal
-                    putText $
-                      sformat ("Update proposal submitted, upId: "%hashHexF) id
+            submitUpdateProposal' sendActions ccPeers ss updateProposal
+
+proposeUnlockStakeEpoch ::
+       ( HasConfiguration
+       , HasInfraConfiguration
+       , HasUpdateConfiguration
+       , HasNodeConfiguration
+       , HasCompileInfo
+       )
+    => SendActions AuxxMode
+    -> ProposeUnlockStakeEpochParams
+    -> AuxxMode ()
+proposeUnlockStakeEpoch sendActions ProposeUnlockStakeEpochParams{..} = do
+    CmdCtx{ccPeers} <- getCmdCtx
+    skey <- (!! puseSecretKeyIdx) <$> getSecretKeys
+    let bvm =
+            BlockVersionModifier
+            { bvmScriptVersion     = Nothing
+            , bvmSlotDuration      = Nothing
+            , bvmMaxBlockSize      = Nothing
+            , bvmMaxHeaderSize     = Nothing
+            , bvmMaxTxSize         = Nothing
+            , bvmMaxProposalSize   = Nothing
+            , bvmMpcThd            = Nothing
+            , bvmHeavyDelThd       = Nothing
+            , bvmUpdateVoteThd     = Nothing
+            , bvmUpdateProposalThd = Nothing
+            , bvmUpdateImplicit    = Nothing
+            , bvmSoftforkRule      = Nothing
+            , bvmTxFeePolicy       = Nothing
+            , bvmUnlockStakeEpoch  = Just puseUnlockStakeEpoch
```

Comment:
I would define a `BlockVersionModifier` with all fields set to `Nothing` somewhere and use it here (and probably in the command above).

Reply:
As a matter of fact, I recall that it was defined somewhere. Good idea to use it.